crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -23,6 +23,9 @@ pub(crate) struct Epoch {
     pub stored: QueryableEpochInfo,
 }
 
+/// Operation of the Sui network is temporally partitioned into non-overlapping epochs,
+/// and the network aims to keep epochs roughly the same time length as each other.
+/// During a particular epoch, the set of validators participating in the network is fixed.
```

Comment:
nit: we might as well specify the other things that are fixed during the epoch as well, e.g.:

- Reference gas price
- Protocol version